extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -50,18 +69,32 @@ class AmpAccordion extends AMP.BaseElement {
       header.setAttribute('role', 'tab');
       content.classList.add('-amp-accordion-content');
       content.setAttribute('role', 'tabpanel');
-      content.setAttribute(
-          'aria-expanded', section.hasAttribute('expanded').toString());
       let contentId = content.getAttribute('id');
       if (!contentId) {
         contentId = this.element.id + '_AMP_content_' + index;
         content.setAttribute('id', contentId);
       }
+      if (this.currentState_[contentId]) {
+        section.setAttribute('expanded', '');
+        content.setAttribute('aria-expanded', 'true');
+      } else if (this.currentState_[contentId] == false) {
+        section.removeAttribute('expanded');
+        content.setAttribute('aria-expanded', 'false');
+      } else {
+        content.setAttribute(
+            'aria-expanded', section.hasAttribute('expanded').toString());
+      }
       header.setAttribute('aria-controls', contentId);
       header.addEventListener('click', boundOnHeaderClick_);
     });
   }
 
+  getSessionStorageKey() {
```

Comment:
annotate and also this should be a private function so it shoud be 
getSessionStorageKey_